bin/ovirt-ha-broker.in

```diff
@@ -1,4 +1,4 @@
-#!@PYTHON@
+#!/bin/python3
```

Comment:
I guess the makefile will replace @PYTHON@.
If that's not needed anymore, we should remove the replacement in the makefile + rename the file to not have a .in suffix